La-Mula-App-main/src/main/kotlin/com/lamulaapp/domain/Order_Status.kt

```diff
@@ -0,0 +1,32 @@
+package com.lamulaapp.domain
+
+import jakarta.persistence.*
+import java.util.*
+
+@Entity
+@Table(name = "order_statuses")
+data class Order_Status(
+    @Id
+    @GeneratedValue(strategy = GenerationType.UUID)

```

Comment:
Recuerda que todos los GenerationType son de tipo IDENTITY ya que la base de datos genera este valor.

Realiza este cambio en todos los lugares necesarios.

```suggestion
    @GeneratedValue(strategy = GenerationType.IDENTITY)

```